<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3387" started-at="2021-04-01T15:28:43 IST" finished-at="2021-04-01T15:28:47 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3387" started-at="2021-04-01T15:28:43 IST" finished-at="2021-04-01T15:28:47 IST">
      <class name="com.vm.SeAutomationTraining.TakeScreensotIfTestFails">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.vm.SeAutomationTraining.TakeScreensotIfTestFails@4e7dc304]" name="openBrowser" is-config="true" duration-ms="3064" started-at="2021-04-01T15:28:40 IST" finished-at="2021-04-01T15:28:43 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="test()[pri:0, instance:com.vm.SeAutomationTraining.TakeScreensotIfTestFails@4e7dc304]" name="test" duration-ms="2772" started-at="2021-04-01T15:28:43 IST" finished-at="2021-04-01T15:28:46 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [The Interne] but found [The Internet]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [The Interne] but found [The Internet]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.vm.SeAutomationTraining.TakeScreensotIfTestFails.test(TakeScreensotIfTestFails.java:16)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="getScreenshotIfTestFails(org.testng.ITestResult)[pri:0, instance:com.vm.SeAutomationTraining.TakeScreensotIfTestFails@4e7dc304]" name="getScreenshotIfTestFails" is-config="true" duration-ms="583" started-at="2021-04-01T15:28:46 IST" finished-at="2021-04-01T15:28:47 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test status=FAILURE method=TakeScreensotIfTestFails.test()[pri:0, instance:com.vm.SeAutomationTraining.TakeScreensotIfTestFails@4e7dc304] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getScreenshotIfTestFails -->
        <test-method status="PASS" signature="killsession()[pri:0, instance:com.vm.SeAutomationTraining.TakeScreensotIfTestFails@4e7dc304]" name="killsession" is-config="true" duration-ms="86" started-at="2021-04-01T15:28:52 IST" finished-at="2021-04-01T15:28:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killsession -->
      </class> <!-- com.vm.SeAutomationTraining.TakeScreensotIfTestFails -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
